src/session.ts

```diff
@@ -82,22 +88,42 @@ export const get = async (sessionId: string): Promise<ISession> => {
 };
 
 export const commit = async (
+  req: express.Request,
   serviceId: string,
-  sessionId: string
+  sessionId: string,
+  options: IEveryAuthOptions
 ): Promise<{ identityId: string; tenantId: string; userId: string }> => {
   const profile = await getAuthedProfile();
   const baseUrl = getSessionUrl(profile);
 
   debug(`${sessionId}: committing`);
 
-  // Start the commit process
+  // Invoke callback if provided
+  if (options.onAuthorized) {
+    let result = await superagent
+      .get(`${baseUrl}/session/${sessionId}/`)
+      .set('User-Agent', EveryAuthVersion)
+      .set('Authorization', `Bearer ${profile.accessToken}`);
+    const ctx = {
+      serviceId,
+      tags: result.body.tags,
+    };
+    await options.onAuthorized(req, ctx);
```

Comment:
```
  debug(`${sessionId}: invoking onAuthorized event`);
```

Also, any desire to allow onAuthorized to abort, or should it throw an exception if desires?

Reply:
I am leaving it at exception for now, we can add a first class support later once we understand what's needed.